src/foam/box/HTTPBox.js

```diff
@@ -26,8 +26,9 @@ foam.CLASS({
   ],
 
   imports: [
+    'generator? as ctxGenerator',
```

Comment:
Boxes are not generally interoperable with existing systems.  Why do we need the ability to inject other serializations into boxes?